Libraries/StyleSheet/StyleSheet.js

```diff
@@ -13,7 +13,16 @@
 
 var StyleSheetRegistry = require('StyleSheetRegistry');
 var StyleSheetValidation = require('StyleSheetValidation');
-var flattenStyle = require('flattenStyle');
+
+/* TODO(brentvatne) docs are needed for this */
+var flatten = require('flattenStyle');
+var PixelRatio = require('PixelRatio');
+
+// Used to compute the constant Stylesheet.Constants.HAIRLINE_WIDTH
```

Comment:
Unfortunate since in the ideal API this would be responsive, but I'll find a workaround.